### PR TITLE
Set Leo default model to Llama 3 8b (25% rollout)

### PR DESCRIPTION
**Pull request by nvonpentz**
Related to https://github.com/brave/brave-core/pull/21398
Blocked on deploying https://github.com/brave/aichat-ops/pull/359 to prod

This sets the default model for 25% of free users to `chat-basic`, which corresponds to [Llama 3 8b](https://github.com/brave/brave-core/blob/c839e1676031a3d155b04b9dbfe49e11b3b8601b/components/ai_chat/core/browser/model_service.cc#L154).

The intention is to start this to progressively roll out this change (25%, 50%, 75%, 100%), so we can ensure our single instance of Llama 3 can handle the increase in traffic.  Based on these [estimations](https://artificialanalysis.ai/?models_selected=llama-3-1-instruct-8b%2Cmixtral-8x7b-instruct), one instance of llama 3 (one gpu) should have higher token throughput than our single instance of mixtral (four gpus), so we expect this to work with two llama instances which will be added in https://github.com/brave/aichat-ops/pull/359.

cc @petemill @LorenzoMinto

Note:
* Chromium version 122.0.6261.57 was selected because that was the first chromium version when 1.63.x was released (see https://bravesoftware.slack.com/archives/C04PX1BUN/p1708629893634639), which is when https://github.com/brave/brave-core/pull/21398 went in.
* I have included all platforms because we want to make this change across all platforms, however this does differ from the BraveAIChatEnabledStudy which applies only to desktop

### CODE REVIEW

File: seed/seed.json
Comment (petemill) [inline seed/seed.json:35]: looks like in this file the versioning is [chrome_major].[brave_version], e.g `122.1.63.0`?

**Comment by petemill** [comment]
I tested running with `--enable-features=AIChat:default_model\/chat-basic` and all seems fine

File: seed/seed.json
Comment (nvonpentz) [reply seed/seed.json]: I updated to use `122.1.63.161` as the min version of the default model change. 1.63.161 seems to be the first 1.63 release. This also adds in a little buffer because https://github.com/brave/brave-browser/issues/34721 was uplifted into 1.62.x.

I specified `122.1.63.160` as the new max version for the previous setting. This may not correlate to an actual release, it's just one patch version behind `122.1.63.161`.

**Review by petemill — APPROVED** [review]

**Comment by kjozwiak** [comment]
@petemill will grab this tomorrow/verify and push it into `production` 👍

File: seed/seed.json
Comment (petemill) [inline seed/seed.json:16]:
```suggestion
                    "parameters": [
                        {
                            "name": "default_model",
                            "value": "chat-basic"
                        }
                    ],
```

**Comment by kjozwiak** [comment]
## `Windows x64` verification(s):

Verification **PASSED** on `Win 11 x64` using the following build(s):

```
Brave | 1.71.118 Chromium: 130.0.6723.70 (Official Build) (64-bit)
-- | --
Revision | d29310b051bde8ba041cd7b4edcb41ce951be12c
OS | Windows 11 Version 23H2 (Build 22631.4317)
```

### `Release` - `BraveAIChatDefaultModelStudy:DefaultLlama`

* launched/re-launched `1.71.118 Chromium: 130.0.6723.70` using `brave.exe --enable-logging=stderr --variations-pr=1225`
* ensured that `pull/1225@02be91adb07839f2668690152f397b0a9ec4c0c2` via `brave://version` under `Active variations`
* ensured that `BraveAIChatDefaultModelStudy:DefaultLlama` via `brave://version`
* ensured that `Llama 3.1 8B` is selected as the default model via `brave://settings/leo-assistant`
* ensured that `Llama 3.1 8B` is selected as the default model via the side panel
* ensured that `Llama 3.1 8B` is being used when using `Brave Leo` when `BraveAIChatDefaultModelStudy:DefaultLlama`

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|---------------|----------------|-----------------
image | image | image | image

### `Release` - `BraveAIChatDefaultModelStudy:DefaultMixtral`

* launched/re-launched `1.71.118 Chromium: 130.0.6723.70` using `brave.exe --enable-logging=stderr --variations-pr=1225`
* ensured that `pull/1225@02be91adb07839f2668690152f397b0a9ec4c0c2` via `brave://version` under `Active variations`
* ensured that `BraveAIChatDefaultModelStudy:DefaultMixtral` via `brave://version`
* ensured that `Mixtral` is selected as the default model via `brave://settings/leo-assistant`
* ensured that `Mixtral` is selected as the default model via the side panel

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|---------------|----------------|-----------------
image | image | image | image

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers.

**Pull request merged**
